src/Bundle/spec/Controller/ResourcesCollectionProviderSpec.php

```diff
@@ -76,7 +76,7 @@ function it_handles_Pagerfanta(
 
         $paginator->setMaxPerPage(5)->shouldBeCalled();
         $paginator->setCurrentPage(6)->shouldBeCalled();
-        $paginator->getCurrentPageResults()->shouldBeCalled();
+        $paginator->getCurrentPageResults()->willReturn([]);
```

Comment:
Why did you change this line?

Reply:
When I originally did this the tests failed because of the lack of a return, most of the Pagerfanta package has return types declared in the 3.x branch.